### PR TITLE
Bug await fix

### LINKED ISSUE #1: Fix Await Issue

### PR DESCRIPTION
**Pull request by DragosRotaru**
#1

### CODE REVIEW

File: api/src/index.ts [hunk 1]
```diff
@@ -186,38 +192,44 @@ console.log(provinces[6]);
...
-  var lng1=latAndLng1.lng;
-  const latAndLng2: {lat:number, lng:number} = await schoolcoordFinder("bc","el",0);
-  var lat2=latAndLng2.lat;
-  var lng2=latAndLng2.lng;
-  console.log(lat2+" "+ lng2);
-  const distance:number= await distanceCalculator(lat1,lng1,lat2,lng2);
+  var lat1 = latAndLng1.lat;
+  var lng1 = latAndLng1.lng;
+  // Pretend Fix
+  // Pretend I Fix Again
```
Comment (shreshthmehra) [inline api/src/index.ts]: It's really not fixed.

File: api/src/index.ts [hunk 2]
```diff
@@ -186,38 +192,44 @@ console.log(provinces[6]);
...
+  var lat2 = latAndLng2.lat;
+  var lng2 = latAndLng2.lng;
+  console.log(lat2 + " " + lng2);
+  const distance: number = await distanceCalculator(lat1, lng1, lat2, lng2);
   console.log(distance);
-
-  
-})();
\ No newline at end of file
+})();
```
Comment (shreshthmehra) [inline api/src/index.ts]: Great job.

File: api/src/index.ts (hunk 1 above)
Comment (DragosRotaru) [reply api/src/index.ts]: You're right! damn

File: api/src/index.ts (hunk 2 above)
Comment (DragosRotaru) [reply api/src/index.ts]: Thx

**Pull request merged**